tests/infrastructure/src/test/java/org/mule/test/infrastructure/SpanTestHierarchyTestCase.java

```diff
@@ -80,6 +80,117 @@ public class SpanTestHierarchyTestCase extends AbstractMuleTestCase {
   @Rule
   public ExpectedException expectedException = none();
 
+
+  @Test
+  public void testWhenTraceStateKeyAssertedAndNotPresentAssertionShouldFail() {
```

Comment:
We can rename this test to: testWhenTraceStateKeyIsExpectedButIsNotPresentAssertionShouldFail